Dockerfile

```diff
@@ -33,7 +33,8 @@ RUN set -x \
  && /fossology/utils/fo-installdeps -e -y \
  && $_cleanup
 RUN curl -sS https://getcomposer.org/installer | php && \
-    mv composer.phar /usr/local/bin/composer
+	mkdir /tmp/composer && \
+    mv composer.phar /tmp/composer/composer
```

Comment:
You points wrt noexec are valid so I agree we should find a better place to host it. 
I didn't want to perturbate your process too much, but indeed downloading it in the fossology tree should bring what we need and solve the issues you mentioned. I'll propose a new patch for that.